drivers/display/ilitek/ili9163c/ili9163c.c

```diff
@@ -440,8 +440,8 @@ static const struct ili9163c_quirks ili9163c_quirks = {
 			},                                                                         \
 		.pixel_format = DT_INST_PROP(n, pixel_format),                                     \
 		.rotation = DT_INST_PROP(n, rotation),                                             \
-		.x_resolution = ILI9163C_X_RES,                                                    \
-		.y_resolution = ILI9163C_Y_RES,                                                    \
+		.x_resolution = DT_INST_PROP(n, width),                                                    \
+		.y_resolution = DT_INST_PROP(n, height),                                                    \
```

Comment:
Please line up the `\`

Reply:
Resolved in [0c6afee](https://github.com/catie-aq/zephyr_ilitek-ili9163c/pull/8/commits/0c6afee60a897ead16390962a16dc220d7aa0dbe).